src/course-outline/data/api.js

```diff
@@ -30,6 +30,12 @@ export const getChapterBlockApiUrl = (courseId, chapterId) => {
   return `${getApiBaseUrl()}/xblock/block-v1:${formattedCourseId}+type@chapter+block@${formattedChapterId}`;
 };
 
+export const getSequentialBlockApiUrl = (courseId, unitId) => {
+  const formattedCourseId = courseId.split('course-v1:')[1];
+  const formattedUnitId = unitId.split('@').slice(-1)[0];
+  return `${getApiBaseUrl()}/xblock/block-v1:${formattedCourseId}+type@sequential+block@${formattedUnitId}`;
+};
+
```

Comment:
Actually, we don't need these functions at all. We are basically reconstructing the item id. :sweat_smile: 

Please remove this function along with `getChapterBlockApiUrl` and `getCourseBlockApiUrl` and replace them with `getCourseItemApiUrl`.

Reply:
@mavidser You missed to remove `getCourseBlockApiUrl` and I missed to catch it. Anyways, I'll remove it in the combined PR. No action is required, just wanted to let you know.